pandas/tools/tile.py

```diff
@@ -191,7 +197,11 @@ def _bins_to_cuts(x, bins, right=True, labels=None, retbins=False,
     ids = bins.searchsorted(x, side=side)
 
```

Comment:
check all the valid possibilities for ``errors`` and raise otherwise (IOW, if you pass a bad value should raise an informative message)

```
if errors not in ['raise', 'drop']:
   raise ValueError("invalid value for errors paramters, valid are: raise, drop")
```